### ISSUE TITLE
[Fix] Manual input triggering

### ISSUE THREAD

**Issue by FlyingWraptor** [passbolt/passbolt_browser_extension#79]
## Manual input triggering
* Passbolt Version: commit 37369540952fd0046cd400f693ca04ac5b37a188 (HEAD, tag: v2.10.0)
* Platform and Target:
-- Browser: Microsoft Edge (Insider Beta) 76.0.182.6 (Official build) dev (64-bit)

### What you did
Logging into a website that uses vue.js (R6stats.com is where I bumped into it) has the issue of the input field event not being triggered by the extension

### What happened
The values will be entered, but virtual dom frameworks do not get the memo of it being there. Resulting in it returning it as 'null' or empty

### What you expected to happen
It to work?

FIX:
![image](https://user-images.githubusercontent.com/38633741/59542257-321ed280-8f05-11e9-9650-a135bab90d8c.png)
```javascript
var event = new Event('input', {
    'bubbles': true,
    'cancelable': true
});

document.querySelector('#login_username').dispatchEvent(event);
document.querySelector('#login_password').dispatchEvent(event);
```

Thanks to Manno from R6Stats for checking this out for me.

**Comment by stripthis** [comment]
@FlyingWraptor thanks for the report, we'll test and include this fix in the next release.

**Comment by stripthis** [comment]
@FlyingWraptor out of curiosity, you are saying in your report that you use edge. How do you load the extension? Are there other functionalities that do not work (like clipboard or download a public key file)?

**Comment by FlyingWraptor** [comment]
@stripthis Great question. Honestly everything works as if it's a chrome extension.
I enabled the 'Allow extensions from other stores.':
![image](https://user-images.githubusercontent.com/38633741/59550942-31705580-8f72-11e9-9290-cdb3918ee109.png)

Which allows the installation of Chrome extensions (or any chromium extension) from their respective stores. So I'm running the 2.10.1 version of Passbolt from the Chrome store:
![image](https://user-images.githubusercontent.com/38633741/59550949-4947d980-8f72-11e9-828b-2617a0e69ecf.png)
![image](https://user-images.githubusercontent.com/38633741/59550952-5bc21300-8f72-11e9-8ec8-cc205e928de6.png)

But I'm positive you already knew that.

As for clipboard it works perfectly fine, remembering passwords. Auto-fill, they all work just like the Chrome extension does (haven't seen any broken stuff yet).

"Downloading a public key file" I don't really get. How am I supposed to do that?

**Comment by stripthis** [comment]
@FlyingWraptor thanks for the explanation.

> "Downloading a public key file" I don't really get. How am I supposed to do that?

If you go to your profile and key inspector (https://yourdomain.com/app/settings/keys ) you can download your keys.

**Comment by FlyingWraptor** [comment]
@stripthis Oh I thought you meant something in the extension. No that works perfectly fine as well. Good job ladds

**Comment by cedricalfonsi** [comment]
Fixed with v2.11.0
